data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	He shows you to his map again, this time pointing to a nearby Arachi world. "Once cracked down some aspects of Yottrite, the twins have, an expert in ship making, we will need," he says. "On <destination>, prepared one of their representatives, House Mallob has. Bring them here, you must."`
+			`	You're about to head back to your ship when he hands you 200,000 credits. "Nearly forgot to pay you for this last service, I did."`
+				accept
+	on accept
+		payment 200000
+
+
+
+mission "Coalition Yottrite 7"
+	name "Master Shipwright"
+	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."
+	passengers 1
+	landing
+	to offer
+		has "Coalition Yottrite 6: done"
+	source "Market of Gupta"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`You're immediately hailed by House Mallob when your ship is descending the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive, castle-like structure, which appears to be made out of a similar material to that of Arach ships, sitting at the base of a mountain.`
+			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`
+			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`
+			`	The Arach takes a translation device out of his pocket and formally greets you, "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
+			choice
+				`	"Are you the shipwright I'll be taking to Lugglop?"`
+					goto servant
+				`	"What's with this place? Why are there so many buildings all over?"`
+					goto property
+			label servant
+			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as a chauffeur, I work."`
+				goto mistress
+			label property
+			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
+			label mistress
+			`	He asks that you enter the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`
+			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`
```

Comment:
```suggestion
			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arach vessel, though you still don't know what the extra wheels are for.`
```

Reply:
I rewrote the part since it was frankly a mess, so this section is no longer an issue since it doesn't exist. if you spot any errors on the new stuff do point it out